### DISCUSSION TITLE
Smarter tab closing

### DISCUSSION THREAD

**Discussion by rawtaz** [ViennaRSS/vienna-rss#1070] [Ideas]
It would be great if when closing a tab, the tab that had focus before the current tab, regains focus when the current tab is closed.

**Comment by barijaona** [comment]
How about adopting the Safari way of managing tabs ? ie. open new tabs near the active tab instead of systematically at the right end of the tab bar.

**Comment by rawtaz** [comment]
Bumping this as Vienna 3.5.7 still doesn't "backtrack" focus when you close a tab.

**Comment by TAKeanice** [comment]
This will be done after the new webkit-based browser is there :) See #1312
